torch/testing/_internal/common_methods_invocations.py

```diff
@@ -986,7 +1035,8 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
                                 dtypes=[torch.cdouble], active_if=IS_WINDOWS),
                        SkipInfo('TestGradients', 'test_inplace_grad',
                                 dtypes=[torch.cdouble], active_if=IS_WINDOWS),
-                   )),
+                   ),
+                   aliases=['arccos', ]),
```

Comment:
Nit: "aliases" seems like good metadata to appear early in the parameters initializing the OpInfo. This is fine but maybe put it under the name?

Reply:
Double nit: make it a tuple since it's an immutable iterable